lib/components/app/responsive-webapp.js

```diff
@@ -201,28 +204,20 @@ class RouterWrapper extends Component {
                 '/stop',
                 '/stop/:id'
               ]}
-              render={() => <WebappWithRouter {...this.props} />}
+              render={() => withLoggedInUser(<WebappWithRouter {...this.props} />)}
             />
             <Route
               path={'/account/welcome'}
               component={(routerProps) => {
                 const props = this._combineProps(routerProps)
-                return (
-                  <Auth0Wrapper>
-                    <UserAccountScreen skipIfExistingUser {...props} />
-                  </Auth0Wrapper>
-                )
+                return withLoggedInUser(<UserAccountScreen skipIfExistingUser {...props} />)
```

Comment:
Seems like since this component is wrapped twice that this wrapper could be put directly in the user-accounts-screen.js file.

Reply:
UserAccountScreen now needs a valid `state.otp.user.loggedInUser` which is "provided" by withLoggedInUser.